src/components/screens/dashboard/dashboard-screen.tsx

```diff
@@ -1,19 +1,27 @@
-import { useAtom } from "jotai";
+import { useEffect } from 'react';
+import strings from "../../../localization/strings";
+import { localeAtom } from "../../../atoms/localeAtom"; // Make sure the path is correct
 import { authAtom } from "../../../atoms/auth";
+import { useAtom } from "jotai";
 
 /**
  * Dashboard screen component
- * 
  */
 const DashboardScreen = () => {
+  const [locale] = useAtom(localeAtom);
+  
   const [auth] = useAtom(authAtom);
 
+  useEffect(() => {
+    strings.setLanguage(locale);
+  }, [locale]);
```

Comment:
This works but will become problematic if we need to do this in every component. See the OSS atoms/language.ts you could use this file to replace you r current localeAtom.ts and this will take care of setting the language without having to use the useEffect in this way.

Reply:
Modified dashboard to use languageAtom